src/sysfs.c

```diff
@@ -594,16 +628,18 @@ void sysfs_reset(void)
 	list_erase(&slave_list);
 	list_erase(&cntnr_list);
 	list_erase(&enclo_list);
+	list_erase(&pci_slots_list);
 	list_erase(&slots_list);
 }
 
 void sysfs_scan(void)
 {
 	_scan_enclo();
 	_scan_cntrl();
-	_scan_slots();
+	_scan_pci_slots();
 	_scan_block();
 	_scan_raid();
+	_scan_slots();
```

Comment:
If possible, then ledmon should not generate slot_list

Reply:
I will work on that and send patchset with fix.